"force" flag does not work with bootstrap command

The --force=true flag works as expected with the main command:
```
goagen main --force=true  -d github.com/goadesign/goa-cellar/design
main.go
account.go
bottle.go
health.go
js.go
public.go
swagger.go
```

It does not work when used with the bootstrap command:
```
goagen bootstrap --force=true  -d github.com/goadesign/goa-cellar/design
exit status 2
flag provided but not defined: -force
Usage of app:
  -design string

  -notest

  -out string

  -pkg string
            (default "app")
  -version string

panic: flag provided but not defined: -force

goroutine 1 [running]:
flag.(*FlagSet).Parse(0xc8201d1de8, 0xc82000a1f0, 0x4, 0x4, 0x0, 0x0)
           /usr/local/go/src/flag/flag.go:913 +0x104
github.com/goadesign/goa/goagen/gen_app.Generate(0x0, 0x0, 0x0, 0x0, 0x0)
           /Users/aalkhatib/go/src/github.com/goadesign/goa/goagen/gen_app/generator.go:37 +0x2c8
main.main()
           /Users/aalkhatib/go/src/github.com/goadesign/goa-cellar/goagen605012214/main.go:29 +0xa4

goroutine 17 [syscall, locked to thread]:
runtime.goexit()
           /usr/local/go/src/runtime/asm_amd64.s:1696 +0x1

goroutine 23 [syscall]:
os/signal.loop()
           /usr/local/go/src/os/signal/signal_unix.go:22 +0x18
created by os/signal.init.1
           /usr/local/go/src/os/signal/signal_unix.go:28 +0x37
```

However in the help message for bootstrap, force appears as supported:

```
 goagen bootstrap -h
Equivalent to running the "app", "main", "client" and "swagger" commands.

Usage:
  goagen bootstrap [flags]

Flags:
      --force[=false]: overwrite existing files
      --notest[=false]: Prevent generation of test helpers
      --notool[=false]: Prevent generation of cli tool
      --pkg="app": Name of generated Go package containing controllers supporting code (contexts, media types, user types etc.)
      --tool="[API-name]-cli": Name of generated tool
      --tooldir="tool": Name of generated tool directory

Global Flags:
      --debug[=false]: enable debug mode, does not cleanup temporary files.
  -d, --design="": design package import path
  -o, --out=".": output directory
```

--- a/.travis.yml
+++ b/.travis.yml
@@ -1,7 +1,7 @@
 language: go
 go:
-- 1.5.3
-- 1.6.2
+- 1.6.3
+- 1.7.1
 # matrix:
 #   allow_failures:
 #     - go: tip

--- a/goagen/gen_app/generator.go
+++ b/goagen/gen_app/generator.go
@@ -30,6 +30,7 @@ func Generate() (files []string, err error) {
 
 	set := flag.NewFlagSet("app", flag.PanicOnError)
 	set.String("design", "", "")
+	set.Bool("force", false, "")
 	set.StringVar(&outDir, "out", "", "")
 	set.StringVar(&target, "pkg", "app", "")
 	set.StringVar(&ver, "version", "", "")

--- a/goagen/gen_client/generator.go
+++ b/goagen/gen_client/generator.go
@@ -43,6 +43,7 @@ func Generate() (files []string, err error) {
 
 	set := flag.NewFlagSet("client", flag.PanicOnError)
 	set.String("design", "", "")
+	set.Bool("force", false, "")
 	set.StringVar(&outDir, "out", "", "")
 	set.StringVar(&target, "pkg", "client", "")
 	set.StringVar(&toolDir, "tooldir", "tool", "")

--- a/goagen/gen_main/generator.go
+++ b/goagen/gen_main/generator.go
@@ -95,9 +95,7 @@ func (g *Generator) Generate() (_ []string, err error) {
 	err = g.API.IterateResources(func(r *design.ResourceDefinition) error {
 		filename := filepath.Join(g.OutDir, codegen.SnakeCase(r.Name)+".go")
 		if g.Force {
-			if err2 := os.Remove(filename); err2 != nil {
-				return err2
-			}
+			os.Remove(filename)
 		}
 		if _, e := os.Stat(filename); e != nil {
 			g.genfiles = append(g.genfiles, filename)

--- a/goagen/gen_swagger/generator.go
+++ b/goagen/gen_swagger/generator.go
@@ -28,6 +28,7 @@ func Generate() (files []string, err error) {
 	set.StringVar(&outDir, "out", "", "")
 	set.StringVar(&ver, "version", "", "")
 	set.String("design", "", "")
+	set.Bool("force", false, "")
 	set.Parse(os.Args[1:])
 
 	if err := codegen.CheckVersion(ver); err != nil {

